src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -274,6 +275,7 @@ def perform_post_processing(
         key="post_processing_maps",
         clock=lambda: pd.Timestamp("2020-04-01"),
         seed=0,
+        index_map=IndexMap(),
```

Comment:
Don't we need to increase this above the default 1_000_000?  I'm assuming this would have broken if we needed to since we already process files with more than a million rows.

Reply:
Good question. Did you test post-processing more than 1 million rows? I *think* because there are no key columns, use_crn will be false on the IndexMap, so we shouldn't get an error.

Reply:
Hmm, good point - I def forgot to test `make_results`. I think it won't matter here for the same reason it didn't matter in pseudopeople...which I still don't entirely understand